plugin/web/components/layout/WdContent.tsx

```diff
@@ -0,0 +1,45 @@
+import { Content } from "antd/lib/layout/layout";
+import React,{ memo, ReactNode } from "react";
+
+import "../../../node_modules/antd/dist/antd.less";
```

Comment:
Please avoid urls like this.
Also, I think except for really special cases, most things in node_modules are accessible through the dep you installed. So this less file should be retrievable by `antd/dist/antd.less`

Reply:
It seems you are doing this in multiple places. But can't you import this less file into your top level component (probably main.tsx) and it works for all? I don't think you need to import it anywhere else after that.